stdlib/3/distutils/dist.pyi

```diff
@@ -1,8 +1,56 @@
 from distutils.cmd import Command
-from typing import Any, Dict, Iterable, Mapping, Optional, Tuple, Type
+from typing import IO, Any, AnyStr, Dict, Iterable, List, Mapping, Optional, Tuple, Type
+
+class DistributionMetadata:
+    def __init__(self, path: Optional[str] = ...): ...
+    name: Optional[str]
+    version: Optional[str]
+    author: Optional[str]
+    author_email: Optional[str]
+    maintainer: Optional[str]
+    maintainer_email: Optional[str]
+    url: Optional[str]
+    license: Optional[str]
+    description: Optional[str]
+    long_description: Optional[str]
+    keywords: Optional[str]
+    platforms: Optional[str]
+    classifiers: Optional[str]
+    download_url: Optional[str]
+    provides: Optional[str]
+    requires: Optional[str]
+    obsoletes: Optional[str]
+    def read_pkg_file(self, file: IO[AnyStr]) -> None: ...
```

Comment:
`file` gets passed to `email.message_from_file()` which only accepts `IO[str]`. (And which should be changed to a protocol at some point, but this is out of scope for this PR.)